elements/pf-dropdown/pf-dropdown.css

```diff
@@ -0,0 +1,117 @@
+:host {
+  display: inline-block;
+  position: relative;
+  --_font-size: var(
+    --pf-c-dropdown__menu-item--FontSize,
+    var(--pf-global--FontSize--md, 1rem)
+  );
+  --_font-weight: var(
+    --pf-c-dropdown__menu-item--FontWeight,
+    var(--pf-global--FontWeight--normal, 400)
+  );
+  --_line-height: var(
+    --pf-c-dropdown__menu-item--LineHeight,
+    var(--pf-global--LineHeight--md, 1.5)
+  );
+  --_color: var(
+    --pf-c-dropdown__menu-item--Color,
+    var(--pf-global--Color--dark-100, #151515)
+  );
+  --_background-color: var(
+    --pf-c-dropdown__menu-item--BackgroundColor,
+    transparent
+  );
+}
+
+:host([disabled]) {
+  color: var(--pf-global--Color--dark-200, #6a6e73);
+}
+
+:host([hidden]),
+[hidden] {
+  display: none !important;
+}
+
+slot[name="trigger"] {
+  cursor: pointer;
+}
+
+pf-dropdown-menu {
+  display: none;
+  position: absolute;
+  top: var(
+    --pf-c-dropdown__menu--Top,
+    calc(100% + var(--pf-global--spacer--xs, 0.25rem))
+  );
+  z-index: var(
+    --pf-c-dropdown__menu--ZIndex,
+    var(--pf-global--ZIndex--sm, 200)
+  );
+  padding-top: var(
+    --pf-c-dropdown__menu--PaddingTop,
+    var(--pf-global--spacer--sm, 0.5rem)
+  );
+  padding-bottom: var(
+    --pf-c-dropdown__menu--PaddingBottom,
+    var(--pf-global--spacer--sm, 0.5rem)
+  );
+  background: var(--pf-c-dropdown__menu--BackgroundColor, #fff);
+  background-clip: padding-box;
+  box-shadow: var(
+    --pf-c-dropdown__menu--BoxShadow,
+    var(
+      --pf-global--BoxShadow--md,
+      0 0.25rem 0.5rem 0rem rgba(3, 3, 3, 0.12),
+      0 0 0.25rem 0 rgba(3, 3, 3, 0.06)
+    )
+  );
+  min-width: 100%;
+  margin: 0;
+}
+
+:host([disabled]) pf-dropdown-menu {
+  pointer-events: none;
+  cursor: not-allowed;
+}
+
+pf-dropdown-menu.show {
+  display: block;
+}
+
+pf-button.disabled::part(button),
+:host([disabled]) ::slotted([slot="trigger"]) {
+  color: var(--pf-theme--color--ui-base, #6a6e73) !important;
```

Comment:
![image](https://github.com/patternfly/patternfly-elements/assets/1466420/101677e6-3817-4cb7-bc88-893c303be9fb)
we need to use patternfly v4 tokens, without exceptions. all the styles have to align with upstream